packages/authorization-server/vitest.config.ts

```diff
@@ -2,10 +2,13 @@ import { defineConfig } from "vitest/config";
 
 export default defineConfig({
   test: {
-    globalSetup: ["./test/vitestGlobalSetup.ts"],
+    globalSetup: ["./test/vitestIntegrationGlobalSetup.ts"],
+    include: ["./test//**/*.test.ts"],
```

Comment:
I think this should be changed to
```suggestion
    include: ["./test/*.test.ts"],
```
so that `pnpm test` only runs tests in the root, and not those in the subdirs, while `pnpm test:api` and `pnpm test:integration` run the tests in the subdirs. Otherwise, tests in the subdirs would be executed both by `pnpm test` and by their specific command, and be run more than once by the CI (as in the last run https://github.com/pagopa/interop-be-monorepo/actions/runs/15180267106/job/42688149582?pr=1898).

When run in the monorepo root, as in the CI, `pnpm test` will still run all tests because of the turborepo config.

I followed this approach for the BFF in https://github.com/pagopa/interop-be-monorepo/pull/1860/files#r2097637996, does this make sense to you?

Reply:
it make sense 👍